website/pages/docs/deployment/ecs.md

```diff
@@ -0,0 +1,379 @@
+                "ec2:GetConsoleOutput",
+                "ec2:GetConsoleScreenshot",
+                "ecr:BatchGetImage",
+                "ecr:GetAuthorizationToken",
+                "ecr:GetDownloadUrlForLayer",
+                "kinesis:Get*",
+                "lambda:GetFunction",
+                "logs:GetLogEvents",
+                "sdb:Select*",
+                "sqs:ReceiveMessage"
+            ],
+            "Resource": "*",
+            "Effect": "Deny"
+        }
+    ]
+}
+```
+
+Replace the `REPLACE_WITH_S3_DESTINATION_BUCKET` placeholder with the name of the S3 bucket you want to use to store the data.
+
+
+This policy will allow the ECS task to write data to the S3 bucket you specified while also ensuring that CloudQuery never has access to any of your data.
+
+
+Using the IAM policy that you just defined in `data-access.json`, you are going to attach it directly to the IAM role that the Fargate Task will use. On top of the custom in-line policy you will also attach the `ReadOnlyAccess` policy and the `AmazonECSTaskExecutionRolePolicy` policy.
+```bash
+# Prior to running the following commands, make sure you have replaced the <REPLACE_TASK_ROLE_NAME> placeholder with the name of the IAM role you created earlier in this step.
+aws iam put-role-policy --role-name <REPLACE_TASK_ROLE_NAME> --policy-name DenyData --policy-document file://data-access.json;
+aws iam attach-role-policy --role-name <REPLACE_TASK_ROLE_NAME> --policy-arn arn:aws:iam::aws:policy/ReadOnlyAccess
+aws iam attach-role-policy --role-name <REPLACE_TASK_ROLE_NAME>  --policy-arn arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy 
+
+```
+
+At this point you have a single IAM role that will be used by the Fargate Task to access the required AWS services.
+
+## Step 6: Register a Task Definition
+A task definition is a blueprint that defines one or more containers that run together on the same host machine. In this step, we will create a task definition for the CloudQuery container.
+
+In the task that you are going to create you will override the default entrypoint and create a custom command that will run the `cloudquery sync` command. This will enable you to store the configuration file as an environment variable, rather than having to create a custom image with the configuration file baked in.
+
+You will also need to pass the CloudQuery configuration file to the container. To do that, you will need to base64 encode the configuration file and pass it as an environment variable to the container.
+
+Create a new file named `task-definition.json` with the following contents:
+```json
+{
+  "containerDefinitions": [
+    {
+      "name": "ScheduledWorker",
+      "image": "ghcr.io/cloudquery/cloudquery:<REPLACE_CQ_CLI_VERSION>",
+      "command": [
+        "/bin/sh",
+        "-c",
+        "echo $CQ_CONFIG| base64 -d  > ./file.yml;/app/cloudquery sync ./file.yml --log-console --log-format json"
+      ],
+      "environment": [
```

Comment:
Same with re `WITH` placeholder. Either we use it in all places or not, but we should be consistent